core/src/main/java/jenkins/util/AntClassLoader.java

```diff
@@ -514,6 +512,12 @@ public void addPathComponent(final File file) {
         pathComponents.addElement(file);
     }
 
+    public void addPathFiles(Collection<File> paths) throws IOException {
+        for (File f : paths) {
+            addPathFile(f);
+        }
+    }
+
```

Comment:
A Jenkins-specific change, not present upstream. Moved from `AntWithFindResourceClassLoader` without modification.